src/app/ui/components/AccordionItem/AccordionItem.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import KeyboardArrowDownRoundedIcon from "@mui/icons-material/KeyboardArrowDownRounded";
+import KeyboardArrowUpRoundedIcon from "@mui/icons-material/KeyboardArrowUpRounded";
+import { useState } from "react";
+
+export type AccordionItemProps = {
```

Comment:
Please, create `types.ts` file with all types you need to export. You can have a separate file for each scope.